src/app/shared/task/task.service.ts

```diff
@@ -108,17 +113,10 @@ export class TaskService {
    * @param ids {Array} Task IDs to update
    * @param data {Object} The data to be applied during update in {field: name} format
    */
-  bulkUpdateTasks(ids: Array<string>, data: any): Observable<ITask> {
+  bulkUpdateTasks(ids: Array<string>, data: any): Observable<Object> {
```

Comment:
use specific type, not Object

Reply:
Thanks, it had issues due to lib versions sync, now it's gone. Fixed.